infrastructure/models.py

```diff
@@ -23,6 +23,22 @@ class Meta:
     def __str__(self):
         return self.name
 
+class OwnerStake(models.Model):
+    """ Percentage that owners own and also relation with PowerPlant and Organization """
+    power_plant = models.ForeignKey(
+        'PowerPlant',
+        models.CASCADE,
+        related_name = 'owner_stake'
+    )
+    owners = models.ForeignKey(
```

Comment:
Can we change this field to to `owner` (since each `OwnerStake` will only point to 1 Organization)?